src/app/cart/cart.component.js

```diff
@@ -60,8 +60,17 @@ class CartController {
     this.$window.location.href = this.sessionService.getRole() === 'REGISTERED' ? 'checkout.html' : 'sign-in.html';
   }
 
-  donationStartDate( month, day ) {
-    return new Date( 0, Number( month ) - 1, Number( day ) );
+  donationStartDate( config ) {
```

Comment:
That was my first idea, but since there is an async http call involved, I wasn't sure on the best way to do it.